.goreleaser.yaml

```diff
@@ -0,0 +1,81 @@
+project_name: apm-lambda-extension
```

Comment:
Migrated to GoReleaser because we don't have ARM runners available in GH Actions workflows by default.